### PR TITLE
add snowballstemmer 1.2.1

### PR DESCRIPTION
**Pull request by chohner**

### CODE REVIEW

File: recipes/snowballstemmer/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+{% set sha256 = "919f26a68b2c17a7634da993d91339e288964f93c274f1343e3bbbe2096e1128" %}
+{% set pkg_name = "snowballstemmer" %}
+
+package:
+  name: {{ pkg_name }}
+  version: {{ version }}
+
+source:
+  fn: {{ pkg_name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/s/{{ pkg_name }}/{{ pkg_name }}-{{ version }}.tar.gz
```
Comment (jakirkham) [inline recipes/snowballstemmer/meta.yaml]: `/s/` -> `/{{ pkg_name[0] }}/`

File: recipes/snowballstemmer/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - {{ pkg_name }}
+
+about:
+  home: http://snowballstem.org
```
Comment (jakirkham) [inline recipes/snowballstemmer/meta.yaml]: PyPI list the repo ( https://github.com/shibukawa/snowball_py ) as the home too.

File: recipes/snowballstemmer/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  url: https://pypi.io/packages/source/{{ pkg_name[0] }}/{{ pkg_name }}/{{ pkg_name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
```
Comment (jakirkham) [inline recipes/snowballstemmer/meta.yaml]: Please add `toolchain`.

File: recipes/snowballstemmer/meta.yaml (hunk 2 above)
Comment (chohner) [inline recipes/snowballstemmer/meta.yaml]: Should I link to the language website somewhere or leave it out completely?

File: recipes/snowballstemmer/meta.yaml (hunk 3 above)
Comment (chohner) [inline recipes/snowballstemmer/meta.yaml]: as a build requirement?

Comment (jakirkham) [inline recipes/snowballstemmer/meta.yaml]: Correct.

Comment (jakirkham) [inline recipes/snowballstemmer/meta.yaml]: It configures how compilation happens.

File: recipes/snowballstemmer/meta.yaml (hunk 2 above)
Comment (jakirkham) [inline recipes/snowballstemmer/meta.yaml]: Would leave it out completely. They seems to be closely related, but different project.

File: recipes/snowballstemmer/meta.yaml (hunk 3 above)
Comment (chohner) [inline recipes/snowballstemmer/meta.yaml]: Afaik there is no compilation, the package is just a [bunch of .py files](https://github.com/shibukawa/snowball_py/tree/master/snowballstemmer).

**Comment by jakirkham** [comment]
Right. Sorry. Was thinking of PyStemmer.

File: recipes/snowballstemmer/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+  run:
+    - python
+
+test:
+  imports:
+    - {{ pkg_name }}
```
Comment (jakirkham) [inline recipes/snowballstemmer/meta.yaml]: Could we please not template this one?

**Comment by jakirkham** [comment]
Thanks @chohner.

**Pull request merged**